tests/utils/test_cloud_resources.py

```diff
@@ -0,0 +1,31 @@
+# coding=UTF-8
+from scout.server.app import create_app
+from scout.server.extensions import cloud_tracks
+
+TEST_TRACKS = [
```

Comment:
Whats wrong with fixtures really? 😊 

I guess this is fine, only used by one test file etc, but the same was true for many of the things in fixtures initially. 🤷

Reply:
Nothing really, I will move it there!